pkgs/tools/networking/rosenpass/tools.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, makeWrapper
+, stdenv
+, coreutils
+, findutils
+, gawk
+, rosenpass
+, wireguard-tools
+}:
+stdenv.mkDerivation {
+  pname = "rosenpass-tools";
+  inherit (rosenpass) version src;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postInstall =
+    let
+      rpDependencies = [
+        coreutils
+        findutils
+        gawk
+        rosenpass
+        wireguard-tools
+      ];
+    in
+    ''
+      install -D $src/rp $out/bin/rp
+      install -D $src/doc/rp.1 $out/share/man/man1/rp.1
+      wrapProgram $out/bin/rp --prefix PATH : ${lib.makeBinPath rpDependencies}
+    '';
+
+  meta = {
+    inherit (rosenpass.meta) homepage license maintainers;
+    description = rosenpass.meta.description + " This package contains `rp`, which is a script that wraps the `rosenpass` binary.";
+  };
```

```suggestion
  meta = rosenpass.meta // {
    description = rosenpass.meta.description + " This package contains `rp`, which is a script that wraps the `rosenpass` binary.";
  };
```
so we make sure that we don't forget anything which might be added in the future